internal/pkg/manifest/worker_svc.go

```diff
@@ -51,9 +57,44 @@ func (s *SubscribeConfig) IsEmpty() bool {
 
 // TopicSubscription represents the configurable options for setting up a SNS Topic Subscription.
 type TopicSubscription struct {
-	Name    *string  `yaml:"name"`
-	Service *string  `yaml:"service"`
-	Queue   SQSQueue `yaml:"queue"`
+	Name    *string        `yaml:"name"`
+	Service *string        `yaml:"service"`
+	Queue   SQSQueueOrBool `yaml:"queue"`
```

Comment:
What do you think of using `SQSQueueOrBool` for `SubscribeConfig` as well? 

I know the queue is always created. But it may feels strange for user to see an unmarshal error if they specify `queue: true` for the service-level queue.

We can also error out or ignore it if we see `queue: false` for that queue.

What do you think?

Reply:
I did that before then I reverted it back because it doesn't make sense to allow `queue: true` if it is a field that cannot be turned off (which is the fact that we are going to ignore `queue: false`). It should be quick for them to figure it out as well once they read the doc.